docs/source/howto/run_codes.rst

```diff
@@ -362,6 +359,21 @@ At the end, you receive a confirmation, with the *PK* and the *UUID* of your new
         prepend_text: conda activate environment-name
 
     Note that the configuration is not complete but only shows the relevant lines.
+    For the `conda activate` statement to work, it needs to be properly initialized in the shell in which the job is executed.
+
+    This can be achieved by configuring the ``shebang`` property of the ``Computer`` to ``#!/bin/bash -l``.
+    This ensures that the submission script uses a login shell which initializes conda properly.
+
+    If for some reason your submission script cannot use a login script, the following ``prepend_text`` can be used instead:
```

Comment:
```suggestion
    If the submission script should not use a login shell (e.g. because that sources other dotfiles that are unnecessary), the following ``prepend_text`` can be used instead:
```